cmd/loopd/daemon.go

```diff
@@ -62,7 +62,7 @@ func daemon(config *config) error {
 	looprpc.RegisterSwapClientServer(grpcServer, &server)
 
 	// Next, start the gRPC server listening for HTTP/2 connections.
-	logger.Infof("Starting gRPC listener")
+	loopdLog.Infof("Starting gRPC listener")
```

Comment:
In other packages we use as a variable name just `log`. I see that we also need to instantiate loggers for other packages in `loopd`, but that is just instantiate, pass in and done. I would also just use `log` here.